src/pages/where-we-work.tsx

```diff
@@ -0,0 +1,64 @@
+import { graphql } from 'gatsby'
+import React, { FunctionComponent } from 'react'
+import SimpleLayout from '../layouts/Simple'
+
+interface Props {
+  data: {
+    allContentfulDataGeoRegion
+    allContentfulDataGeoRegionSubRegion
+  }
+}
+
+const WhereWeWorkPage: FunctionComponent<Props> = ({ data }) => {
+  const pageContext = {
+    pageTitle: 'TODO',
+    siteTitle: 'TODO',
+  }
+
+  return (
+    <SimpleLayout pageContext={pageContext}>
+      {/* page header */}
+      <header>
+        <h1>Where We Work</h1>
+      </header>
+
+      <ul>
```

Comment:
Nit (optional): some basic styling would be great! Tailwind has the `list-disc` class to help you out.